pkg/ast/ast_root_operation_type_definition.go

```diff
@@ -92,7 +93,9 @@ func (d *Document) ImportRootOperationTypeDefinition(name string, operationType
 	operationTypeDefinition := RootOperationTypeDefinition{
 		OperationType: operationType,
 		NamedType: Type{
-			Name: d.Input.AppendInputBytes(nameBytes),
+			Name:     d.Input.AppendInputBytes(nameBytes),
+			TypeKind: TypeKindNamed,
+			OfType:   -1,
```

Comment:
I still think we should reuse the constant we already have for this magic number :-)